kmod/patch/kpatch-patch.h

```diff
@@ -40,7 +40,7 @@ struct kpatch_patch_dynrela {
 	char *name;
 	char *objname;
 	int external;
-	int addend;
+	long addend;
```

Comment:
What is this solving?

Reply:
Yes, sorry, for this and some other comments, I did the review using an outdated version of master.
Sorry for the noise.

Reply:
Thank for the valuable suggestions, I will post v2 sooner.